assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -88,12 +88,7 @@ export default function GA4SettingsControls( {
 		select( CORE_SITE ).getDocumentationLinkURL( 'ga4' )
 	);
 
-	const isGA4Connected = useSelect( ( select ) =>
```

Comment:
I noticed this was duplicated as well after my initial review, thanks for consolidating 👍